.gitignore

```diff
@@ -21,3 +21,12 @@ smart_control/learning/
 smart_control/simulator/videos
 smart_control/refactor/data/
 smart_control/refactor/experiment_results/
+smart_control/learning/
+.vscode/
```

Comment:
we do have some .vscode settings to be shared across all developers, but if there are user-specific files you would like to ignore, maybe we can use some more granular references here